tests/helpers_test.cc

```diff
@@ -279,38 +293,38 @@ TEST(helpers, containers_initialize) {
   ASSERT_FALSE(createClient(cfg_dir, apps_cfg).dockerAppsChanged());
 
   // Add a new app
-  apps_cfg["docker_apps"] = "app1";
+  apps_cfg["compose_apps"] = "app1";
 
   ASSERT_TRUE(createClient(cfg_dir, apps_cfg).dockerAppsChanged());
 
   // No apps configured, but one installed:
-  apps_cfg["docker_apps"] = "";
+  apps_cfg["compose_apps"] = "";
   boost::filesystem::create_directories(apps_root / "app1");
   ASSERT_TRUE(createClient(cfg_dir, apps_cfg).dockerAppsChanged());
 
   // One app configured, one app deployed
-  apps_cfg["docker_apps"] = "app1";
+  apps_cfg["compose_apps"] = "app1";
   boost::filesystem::create_directories(apps_root / "app1");
   ASSERT_FALSE(createClient(cfg_dir, apps_cfg).dockerAppsChanged());
 
   // Docker app parameters enabled
-  apps_cfg["docker_app_params"] = (cfg_dir / "foo.txt").native();
-  Utils::writeFile(cfg_dir / "foo.txt", std::string("foo text content"));
-  ASSERT_TRUE(createClient(cfg_dir, apps_cfg).dockerAppsChanged());
-
-  // Store the hash of the file and make sure no change is detected
-  auto client = createClient(cfg_dir, apps_cfg);
-  client.storeDockerParamsDigest();
-  ASSERT_FALSE(client.dockerAppsChanged());
-
-  // Change the content
-  Utils::writeFile(cfg_dir / "foo.txt", std::string("foo text content changed"));
-  ASSERT_TRUE(client.dockerAppsChanged());
-
-  // Disable and ensure we detect the change
-  apps_cfg["docker_app_params"] = "";
-  ASSERT_TRUE(createClient(cfg_dir, apps_cfg).dockerAppsChanged());
-  ASSERT_FALSE(boost::filesystem::exists(cfg_dir / ".params-hash"));
+//  apps_cfg["docker_app_params"] = (cfg_dir / "foo.txt").native();
+//  Utils::writeFile(cfg_dir / "foo.txt", std::string("foo text content"));
+//  ASSERT_TRUE(createClient(cfg_dir, apps_cfg).dockerAppsChanged());
+
+//  // Store the hash of the file and make sure no change is detected
+//  auto client = createClient(cfg_dir, apps_cfg);
+//  client.storeDockerParamsDigest();
+//  ASSERT_FALSE(client.dockerAppsChanged());
+
+//  // Change the content
+//  Utils::writeFile(cfg_dir / "foo.txt", std::string("foo text content changed"));
+//  ASSERT_TRUE(client.dockerAppsChanged());
+
+//  // Disable and ensure we detect the change
+//  apps_cfg["docker_app_params"] = "";
+//  ASSERT_TRUE(createClient(cfg_dir, apps_cfg).dockerAppsChanged());
+//  ASSERT_FALSE(boost::filesystem::exists(cfg_dir / ".params-hash"));
```

Comment:
is this logic breaking your tests or something?

Reply:
It's the docker app manager specific tests, that pacman type is not used anyway, so I just comment them, I can either remove them or create another test and initialize client with the docker app manager so these tests will succeed?

Reply:
oh - i see. a new test would probably be cleaner anyway. thanks

Reply:
added a new test.